Header header

int16 agent_no
int32 id
time obs_stamp
int32 keyframe_id
int16 vehicle_id
int16 object_tag
sensor_msgs/PointCloud2 cloud
geometry_msgs/Pose transform
